documentation/src/main/asciidoc/topics/proc_enabling_metrics.adoc

```diff
@@ -0,0 +1,44 @@
+[id='configure_metrics']
+= Enabling {brandname} Metrics
+{brandname} is compatible with the Eclipse MicroProfile Metrics API and can
+generate gauge and histogram metrics.
+
+* {brandname} metrics are provided at the `/base` and `/vendor` scopes and are
```

Comment:
{brandname} metrics are provided at the 'vendor' scope. Also, JVM-related metrics are provided in the 'base' scope.